README.md

```diff
@@ -1,3 +1,56 @@
-# Backend app
+# Music Library
+
+This is an app designed to perform CRUD requests on a locally-hosted PostgreSQL database. 
+
```

Comment:
I would be good if you could mention the dependencies used in the project i.e. express, postgres, nodemon, mocha, etc. It's also good to distinguish between dependencies and dev dependencies